src/shared/ui/CalendarCard.tsx

```diff
@@ -1,11 +1,62 @@
-import {ReactElement} from "react";
-import {ICardProps} from "../types";
+import { ReactElement } from "react";
+import { ICardProps } from "../types";
+import Svg from "./Svg";
 
-export function CalendarCard({ date, weekDay }: ICardProps): ReactElement {
+interface ICalendarCard {
+  date: ICardProps;
+  isStartDate: boolean;
+  isEndDate: boolean;
+  isBetweenDate: boolean;
+  onClickDate?: (date: ICardProps) => void;
+  onCloseCalendarFilter: () => void;
+}
+
+export function CalendarCard({
+  date,
+  isStartDate,
+  isEndDate,
+  isBetweenDate,
+  onClickDate,
+  onCloseCalendarFilter,
+}: ICalendarCard): ReactElement {
   return (
-    <div className="flex flex-col mr-8 shrink-0 max-w-[27px] cursor-pointer">
-      <p className="text-[22px] font-semibold text-text-black text-center">{date}</p>
-      <p className={`text-[13px] font-medium mt-2 text-text-black text-center uppercase ${weekDay === "СБ" || weekDay === "ВС" ? "text-text-red" : ""}`}>{weekDay}</p>
+    <div className="relative flex flex-col mt-[29px]">
+      {isEndDate ? (
```

Comment:
add isSelectedDate prop and use it here like this: (isEndDate || isSelectedDate) && (...Svg).